tests/test_clients.py

```diff
@@ -35,3 +36,53 @@ def test_close_without_clients(hlwm):
 def test_close(hlwm, running_clients_num):
     hlwm.create_clients(running_clients_num)
     hlwm.call('close')
+
+
+@pytest.mark.filterwarnings("ignore:tostring")
+@pytest.mark.parametrize("urgent", [True, False])
+def test_urgent_on_start(hlwm, x11, urgent):
+    command = []
+    if urgent:
+        # prepent urgent command
```

Comment:
Typo ("prepend")

Reply:
Luckily, I got rid of this part by using python-xlib directly.